indico/modules/events/static/offline.py

```diff
@@ -366,7 +367,7 @@ def _get_session(self, session):
         pdf = get_session_timetable_pdf(session, tz=self._display_tz)
         self._add_pdf(session, 'sessions.export_session_timetable', pdf)
 
-        self._add_file(get_session_ical_file(session), 'sessions.export_ics', session)
+        self._add_file(session_to_ical(session), 'sessions.export_ics', session)
```

Comment:
Have you tested these changes? `_add_file` seems to handle both file objects and strings, but I'd like to be sure there are no problems e.g. with implicit str/bytes happening (or failing) somewhere..

Reply:
Tested, it works!